examples/cpp/dds/AdvancedConfigurationExample/arg_configuration.h

```diff
@@ -229,6 +230,8 @@ const option::Descriptor usage[] = {
       "  -p <str> \t--partitions=<str> \tPartitions to match separated by ';'."
       " Single or double quotes required with multiple partitions."
       " With empty string ('') no partitions used. (Default: '')." },
+    { PROFILE, 0, "x", "profile",         Arg::String,
+      "  -x <str> \t--xml-profile \tProfile name to configure DomainParticipant."},
```

Comment:
The argument parser recognizes the `--profile` option and not the `--xml-profile` as it is printed. Please fix this inconsistency.